Library/Formula/conoha-iso.rb

```diff
@@ -0,0 +1,34 @@
+require "language/go"
+
+class ConohaIso < Formula
+  desc "Command wrapper to use ISO of ConoHa."
+  homepage "https://github.com/hironobu-s/conoha-iso"
+  version "0.2.1"
+  url "https://github.com/hironobu-s/conoha-iso/archive/current.tar.gz"
```

Comment:
'current' It is written in the document that you use to download from to, I tried to use actually are running properly.